### PR TITLE
Enhancement/donate button

### PR DESCRIPTION
**Pull request by Irsanarisandy**
Add capability to change text and set link of header button.

### CODE REVIEW

File: app/code/extensions/ExtraConfig.php [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-use SilverStripe\Forms\NumericField;
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\Forms\TreeDropdownField;
 use SilverStripe\AssetAdmin\Forms\UploadField;
-use SilverStripe\Forms\HTMLEditor\HTMLEditorField;
 
-class FooterConfig extends DataExtension
+class ExtraConfig extends DataExtension
 {
-
```
Comment (IshaqIdris) [inline app/code/extensions/ExtraConfig.php:15]: Possibly add a table name to get rid of that error that comes up when you `dev/build`

File: themes/app/templates/Includes/Header.ss [hunk 2]
```diff
@@ -30,8 +30,8 @@
                 <% end_loop %>
             </div>
         </div>
-        <div slot="button" class="main-nav__donate">
-            <button class="btn btn-outline-secondary" type="button">DONATE NOW</button>
+        <div slot="button" class="main-nav__button">
+            <a class="btn btn-outline-secondary" href="$SiteConfig.HeaderButtonLink.Link" role="button">$SiteConfig.HeaderButtonText</a>
```
Comment (IshaqIdris) [inline themes/app/templates/Includes/Header.ss:34]: Nice! I like this line, might have to steal it of you 😉

File: app/code/extensions/ExtraConfig.php (hunk 1 above)
Comment (Irsanarisandy) [reply app/code/extensions/ExtraConfig.php:15]: No need to add table name if it is within Settings page.

File: themes/app/templates/Includes/Header.ss (hunk 2 above)
Comment (Irsanarisandy) [reply themes/app/templates/Includes/Header.ss:34]: 😘

**Pull request closed without merging**